.ci/common.sh

```diff
@@ -2,7 +2,7 @@
 check_platform()
 {
     MACHINE_TYPE=`uname -m`
-    if [ ${MACHINE_TYPE} != 'x86_64' ]; then
+    if [ ${MACHINE_TYPE} != 'x86_64' ] && [ ${MACHINE_TYPE} != 'aarch64' ]; then
```

Comment:
The result of command `uname -m` on macOS + Apple Silicon is `arm64` instead of `aarch64`.

Reply:
We may support macOS/Arm64 host in CI pipeline later. Let's consolidate the listing.

Reply:
The prebuilt executable is built on Aarch64 machine. For machines with Apple Silicon, it needs a simulator like `lima` to run with the prebuilt executable.

Reply:
> The prebuilt executable is built on Aarch64 machine.

Yes, my intention was to consolidate the machine detection listing no matter what operating system is.